tests/KAT_PubKey/PSS.hs

```diff
@@ -470,4 +339,10 @@ pssTests = testGroup "RSA-PSS"
         [ doVerifyTest rsaKeyInt (katZero, vectorInt) ]
     , testGroup "signature key 1024" $ map (doSignTest rsaKey1) (zip [katZero..] vectorsKey1)
     , testGroup "verify key 1024" $ map (doVerifyTest rsaKey1) (zip [katZero..] vectorsKey1)
+    , testGroup "signature key 1025" $ map (doSignTest rsaKey2) (zip [katZero..] vectorsKey2)
+    , testGroup "verify key 1025" $ map (doVerifyTest rsaKey2) (zip [katZero..] vectorsKey2)
+    , testGroup "signature key 1026" $ map (doSignTest rsaKey3) (zip [katZero..] vectorsKey3)
+    , testGroup "verify key 1026" $ map (doVerifyTest rsaKey3) (zip [katZero..] vectorsKey3)
+    , testGroup "signature key 1031" $ map (doSignTest rsaKey8) (zip [katZero..] vectorsKey8)
+    , testGroup "verify key 1031" $ map (doVerifyTest rsaKey8) (zip [katZero..] vectorsKey8)
```

Comment:
While we're at it, perhaps just run through all the test vectors from RSA for the 10 key sizes?
```
# Key  1: 1024 bits
# Key  2: 1025 bits
# Key  3: 1026 bits
# Key  4: 1027 bits
# Key  5: 1028 bits
# Key  6: 1029 bits
# Key  7: 1030 bits
# Key  8: 1031 bits
# Key  9: 1536 bits
# Key 10: 2048 bits
```

Reply:
Initially I did all examples. But module was growing large, so I removed some vectors not really necessary.